src/tests/ftest/pool/bad_query.py

```diff
@@ -39,10 +40,6 @@ def test_query(self):
         handle = handlelist[0]
         expected_for_param.append(handlelist[1])
 
-        infolist = self.params.get("info", '/run/querytests/infoptr/*/')
-        dummy_infoptr = infolist[0]
```

Comment:
Do we know why this wasn't being used? Would it be better to just use this variant? (I'm not familiar with the test)

Reply:
Never mind, I see this was discussed as part of test plan scrubbing

Reply:
I looked at the history. Long time ago, someone thought that this `infoptr` can be passed in to `pool_query()` of DAOS API, although that feature never existed. Then another person disabled the use of this `infoptr` by assigning it to `dummy_infoptr` because they probably found out that the API doesn't accept it. That was 3 years ago. This code has been sitting here since then.